_posts/2016-10-21-sweet-break-before-break-102.md

```diff
@@ -0,0 +1,120 @@
+---
+
+title: 'A sweet break before the break: TGIF (102)'
+author: Robert
+layout: blog-post-tgif
+comments: false
+
+---
+
+Thank Grohl, Gauss, Glob, Galileo and Science it’s Friday! This Friday’s little bit of knowledge, fascination, and fun is curated for you by [Rob](https://twitter.com/gibbiv) :)
+
+## Editors’ Note: A short hiatus
+
+Last week I hopped on a call with [Cecily](https://twitter.com/skeskali), [Gregor](https://twitter.com/gr2m), and [Jenn](https://twitter.com/jennwrites) to discuss the future of TGIF, Hoodie’s lazy Friday afternoon read started by [Lena](https://twitter.com/lrnrd) in [August 2013](http://hood.ie/blog/tgif-hoodies-top-links-of-the-week-1.html). After an hour-long chat with these amazing peeps we decided to take a step back and rethink how Hoodie’s weekly newsletter is created, delivered and promoted.
+
+Some of the things we discussed were TGIF having it’s own platform (separate from the blog), opening each issue up to multiple contributors (one contributor per section), making TGIF more accessible (with an email subscription option), and reaching out to publishers of content we share (to spread awareness).
+
+I’m grateful to be part of this project and look forward to making TGIF one of the most anticipated newsletters in open source, intellect and fun! Now for what you may have missed over the past 2 weeks...
```

Comment:
“in open source, intellect, and fun” might not translate well to ESL readers, I think you could be more concise here